plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -319,18 +322,33 @@ protected String renameTableSql(String catalogName, String remoteSchemaName, Str
     }
```

Comment:
May be the similar change could be applied to renameTableSql, as it uses sp_rename too.